docs/key_concepts.md

```diff
@@ -29,7 +29,7 @@ As you go, you can also evaluate a [IOMD](iomd.md) code chunks pressing `ctrl+en
 
 The Iodide server provides a hosted solution to make edits and save changes to your Iodide notebook on the Web. The server allows for you to create a profile to save your collection of notebooks, track changes made in a central location.
 
-When editing an iodide notebook, the content of your notebook should be saved at a regular interval (currently 30 seconds) as you type. The server has a collection of all the revisions explicitly saved to the during your session. To access the revisions for a notebook, go to the landing page and hover over a notebook of interest. You will see the option `revisions` that you can click, which will take you the history of revisions for that notebook.
+When editing an iodide notebook, the content of your notebook should be saved at a regular interval (currently 30 seconds) as you type. In order to reduce the number of old revisions, server side also prunes the old revisons to one per minute. The server has a collection of all the revisions explicitly saved to the during your session. To access the revisions for a notebook, go to the landing page and hover over a notebook of interest. You will see the option `revisions` that you can click, which will take you the history of revisions for that notebook.
```

Comment:
I think we can skip the extra explanation here, I don't think this is super important from a user's perspective.

```suggestion
When editing an iodide notebook, the content of your notebook should be saved at a regular interval (currently 30 seconds) as you type. The server has a collection of all the revisions explicitly saved to the during your session. To access the revisions for a notebook, go to the landing page and hover over a notebook of interest. You will see the option `revisions` that you can click, which will take you the history of revisions for that notebook.
```

Reply:
sure